toolbox

```diff
@@ -1732,8 +1747,8 @@ while has_prefix "$1" -; do
             assume_yes=true
             ;;
         -h | --help )
-            usage
-            exit
+            help "$2"
```

Comment:
We should also update the `toolbox --help` output and a manual page for the new `toolbox help` command.

Reply:
What do you mean specifically by updating? I would just specify that for invoking help you can use both flags `-h` and `--help`. And yeah, I need to update the man page and create one specifically for the command.

Reply:
I meant that we should mention the new `help` command in the `toolbox --help` output.